packages/next-tinacms-cloudinary/src/handlers.ts

```diff
@@ -95,9 +95,10 @@ async function listMedia(
   try {
     const {
       directory = '""',
-      limit = 500,
+      limit = '500',
       offset,
-    } = req.query as MediaListOptions
+      filesOnly = 'false',
+    } = req.query
```

Comment:
Should we add back `MediaListOptions`

Reply:
Having MediaListOptions caused confusion as they were actually strings, being of type NextApiRequest:
<img width="799" alt="image" src="https://github.com/user-attachments/assets/e88fda4c-dd78-400d-80e5-a15be591bd54">

I had to spend a while figuring out why my conditionals were not working as expected (`filesOnly` was coming in as a `string` and not `boolean`.

If we do want to have MediaListOptions there as a easier way people to track where this query comes from, I can do a type conversion from NextApiRequest to MediaListOptions?